scripts/exgdas_global_marine_analysis_run.sh

```diff
@@ -108,6 +131,13 @@ concatenate_bump 'bump3d'
 clean_yaml var.yaml
 $APRUN_SOCAANAL $JEDI_BIN/soca_var.x var.yaml 2>var.err
 
+
+# increment update for MOM6 iau
+# Make sure the last (most recent) increment file put into socaincr2mom6 as $1
```

Comment:
I don't know what this comment means ...

Reply:
updated